packages/tables/src/styled/StyledCell.ts

```diff
@@ -68,10 +68,6 @@ export const StyledCell = styled.td.attrs<IStyledCellProps>({
   ${props => sizeStyling(props)};
   ${props => props.isTruncated && truncatedStyling};
 
-  &[hidden] {
-    ${hideVisually()}
-  }
```

Comment:
Let's keep this here rather than spreading the logic around to element cell `style`. Use something like `&[hidden] > div` or similar.